roles/edpm_telemetry/tasks/deploy_ceilometer.yml

```diff
@@ -52,20 +52,20 @@
     mode: "{{ item.mode | default('640') }}"
     remote_src: "{{ telemetry_test | default('false') }}"
   with_items:
-    - { 'src': "/var/lib/config-data/merged/ceilometer.conf", "dest": "/var/lib/openstack/config/ceilometer/ceilometer.conf" }
-    - { 'src': "/var/lib/config-data/merged/polling.yaml", "dest": "/var/lib/openstack/config/ceilometer/polling.yaml" }
-    - { 'src': "/var/lib/config-data/merged/ceilometer_agent_compute.json", "dest": "/var/lib/openstack/config/containers/ceilometer_agent_compute.json" }
-    - { 'src': "/var/lib/config-data/merged/ceilometer-agent-compute.json", "dest": "/var/lib/openstack/config/ceilometer/ceilometer-agent-compute.json" }
-    - { 'src': "/var/lib/config-data/merged/ceilometer_agent_ipmi.json", "dest": "/var/lib/openstack/config/containers/ceilometer_agent_ipmi.json" }
-    - { 'src': "/var/lib/config-data/merged/ceilometer-agent-ipmi.json", "dest": "/var/lib/openstack/config/ceilometer/ceilometer-agent-ipmi.json" }
+    - { 'src': "{{edpm_ceilometer_config_src}}/ceilometer.conf", "dest": "{{edpm_ceilometer_config_dest}}/ceilometer/ceilometer.conf" }
```

Comment:
or you could add a /telemetry in here

Reply:
Solved in the default value of the variable container